test/kubernetes_test.go

```diff
@@ -479,8 +840,63 @@ func TestKubernetesIntegrationFallthrough(t *testing.T) {
 	drop 0
     }
 `
-	cases := append(dnsTestCases, dnsTestCasesFallthrough...)
-	doIntegrationTests(t, corefile, cases)
+	doIntegrationTests(t, corefile, dnsTestCasesFallthrough)
+}
+
+func getDynamicEndpoints(Qname, namespace, name string) []dns.RR {
+	rr := []dns.RR{}
+
+	result := strings.Split(getCmdOutput(name, namespace), " ")
+	len_result := len(result)
```

Comment:
> @chrisohaver told me we parse the json from kubectl, here we just string split the output?

It's splitting output produced by the json path filter.

> I don't now what happens if you specify a json path

It depends on what path you give it. In this case, the path will list all IPs for each endpoint.